### ISSUE TITLE
Update hosted arm2pulumi or at least put an info banner there

### ISSUE THREAD

**Issue by mnlumi** [pulumi/pulumi-azure-native#2593]

**Comment by lukehoban** [comment]
Should we instead just get it working with v2?

**Comment by mnlumi** [comment]
@lukehoban Yes, getting it working with v2 is certainly the preferred and ultimate goal. The banner is meant to be an intermediate step while awaiting prioritization of arm2pulumi fixes. cc @mikhailshilkov
